### PR TITLE
Refactor/search css

### PR DESCRIPTION
**Pull request by jasmyn2244**
### What (if any) features are you implementing?
- none
### What (if anything) did you refactor?
- css
### Were there any issues that arose?
- n/a
### Is there anything that you need from your teammate?
- n/a
### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [ ] Are commit messages formatted correctly?

### CODE REVIEW

File: src/Components/Header/Header.scss [hunk 1]
```diff
@@ -7,6 +7,6 @@
 }
 
 .long-logo {
-    width: 70%;
+    width: 64%;
```
Comment (anewb87) [inline src/Components/Header/Header.scss:10]: Some minor adjusting here and there.

File: src/Components/MovieContainer/MovieContainer.js [hunk 2]
```diff
@@ -51,37 +51,43 @@ class MovieContainer extends React.Component {
...
-                </section>
+                    <section className='search-error'>
+                        <h2 className='error-text'>No tomatillos for you!<br />Try a different movie!</h2>
+                    </section>
+                </>
             )
         } else {
             return (
-            <>
-                {this.renderCards(this.state.searchResults)}
```
Comment (Gabby-Recny) [inline src/Components/MovieContainer/MovieContainer.js:76]: Nice sweater Jasmyn

File: src/Components/MovieContainer/MovieContainer.js [hunk 3]
```diff
@@ -91,7 +97,7 @@ class MovieContainer extends React.Component {
             return (
                 <section className='movie-container' >
                     <SearchBar searchMovies={this.searchMovies} />
-                   {this.state.isSearching && this.getSearchMovieCards()}
```
Comment (Gabby-Recny) [inline src/Components/MovieContainer/MovieContainer.js:94]: Sweet flannel Lex. Looking warm!

File: src/Components/MovieContainer/MovieContainer.scss [hunk 4]
```diff
@@ -10,12 +10,43 @@
...
 }
 
 .search-error {
-    position: RELATIVE;
-    left: 90%;
     text-align: center;
     width: 125%;
+}
+
+.main-page-btn-section {
```
Comment (anewb87) [inline src/Components/MovieContainer/MovieContainer.scss:20]: Getting our button where we want it!

**Review by Gabby-Recny — APPROVED** [review]

File: src/Components/SearchBar/SearchBar.scss [hunk 5]
```diff
@@ -5,27 +5,36 @@ form {
...
 .search-bar {
     font-size: 1em;
     color: white;
     text-align: center;
     font-family: 'Raleway', sans-serif;
     border: 0px;
     border-radius: 5px;
-    padding: 4px 30px 4px 30px;
-    background-color: black;
+    padding: 5px 10px;
```
Comment (anewb87) [inline src/Components/SearchBar/SearchBar.scss:20]: Love the look.

**Review by anewb87 — APPROVED** [review]
Smelling that finish line. And it smells like the sweet success of artisan crusty bread.

**Pull request merged**